serverless.ts

```diff
@@ -0,0 +1,41 @@
+import * as dotenv from "dotenv";
+import type { AWS } from "@serverless/typescript";
+dotenv.config();
+
+import { slackbot } from "@functions/slackbot";
+
+const config: AWS = {
+  service: "treeumph",
+  frameworkVersion: "3",
+  plugins: ["serverless-esbuild", "serverless-offline"],
+  provider: {
+    name: "aws",
+    runtime: "nodejs14.x",
```

Comment:
Is this still the most current version that Lambda supports?

Reply:
Actually meant to update this to 16, this is a copy pasta job from an sls example 🤦 Double checked what lambda supports and it can do 18 so have updated to use that now